content/project-framing.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Framing the Project
+home_spotlights:
+  enabled: false
+  weight: 1
+  excerpt: The success of your project largely depends on understanding your nonprofit's project.
+      Before you crack open your IDE and dive into the code, take some time to glean important insights
+      up front, so that you can be more efficient with your time during the hack.
+  home_img:
+    path: images/framing-barn.jpg
+    data_position: center center
+    url: "#"
+layout: page
+---
+
+Congratulations! You've selected your nonprofit project and are ready to get started. Before you crack open your IDE and dive into the code, the first few hours of the jam are most well spent building a deep understanding of the problem your nonprofit is trying to solve. By doing this well, you will have a better defined path for the next 40+ hours. You may also discover unexpected solutions that your nonprofit didn't even know to ask for. After all, they are experts in serving their communities, not the nuances of the latest and greatest technology.
+
+## Interviewing Your Nonprofit
+
+
+
+### Take (Post It) Notes
+A lot is about to be said in the next hour. You cannot possibly remember it all. You also don't know enough yet to understand how what they are saying this moment connects to an insight you'll have in two hours. You could do this in a notebook or shared document, like you're probably used to for school, or you could try something a bit different.
+
+PostIt Notes have become almost [synonymous with brainstorming & creativity](https://www.nngroup.com/articles/post-it-in-ux/) for good reason. They are flexible, tangible, collaborative, and low tech. By taking notes on PostIts, you can quickly connect and organize ideas in a way that both your team and nonprofits can quickly see and contribute to. Jot one idea per note. Come up with a loose color coding (like blue for notes, yellow for ideas, red for pain points, or you're fine just sticking to one color if it is too much to keep track of). Don't worry too much about organizing them at first. You can start by putting them randomly on a wall or even a table. Once the ideas are out there, you can start grouping them. Stick related notes to one another so you can move the group all together.
+
+![PostIt Notetaking from 2014 DSMHack][mypostit] ![PostIt Notetaking. Photo credit: https://medium.com/digital-experience-design/5-tricks-to-master-the-art-of-post-it-notes-fa02f3f5a2e5][postit]
+
+
+### Active & Open Listening
+The focus of this time is to hear what they have to say, not to try to push your own solution or ideas (wait for it, that can come later). By using [active listening](https://www.mindtools.com/CommSkll/ActiveListening.htm) and open interview questions, you'll get much richer information.
+- **Pay attention.** This is not the time to be multi-tasking on your phone.
```

Comment:
"multi-tasking on your phone" might come off as patronizing or be laughed off because they don't think they'll do that. I think you could call them out on more specific things like "setting up your laptop", "bootstrapping your preferred framework". Maybe that's just me 😝

Reply:
I think that is very fair, and I like your suggestions better 😄